packages/client/modules/email/components/SummaryEmail/MeetingSummaryEmail/WholeMeetingSummaryResult.tsx

```diff
@@ -55,16 +57,30 @@ const WholeMeetingSummaryResult = (props: Props) => {
     `,
     meetingRef
   )
-  const atmosphere = useAtmosphere()
-  const {summary: wholeMeetingSummary, team} = meeting
-  const explainerText = team?.tier === 'starter' ? AIExplainer.STARTER : AIExplainer.PREMIUM_MEETING
   useEffect(() => {
     SendClientSideEvent(atmosphere, 'AI Summary Viewed', {
       source: 'Meeting Summary',
       tier: meeting.team.billingTier,
       meetingId: meeting.id
     })
-  }, [])
+  }, [atmosphere, meeting.id, meeting.team.billingTier])
+
+  const {summary: wholeMeetingSummary, team} = meeting
+
+  if (!wholeMeetingSummary) return null
+
+  const renderedSummary = marked(wholeMeetingSummary, {
+    gfm: true,
+    breaks: true
+  }) as string
+  // replace links with styled links
```

Comment:
-1 this is obvious, but instead we should explain why we do this instead of doing a cleaner approach so it serves as a warning not to copy this.